src/modules/landing_target_estimator/KFxyzCoupledMoving.cpp

```diff
@@ -0,0 +1,483 @@
+	_covariance(10, 1) = _covariance(10, 1) + _tmp31 + _tmp45;
+	_covariance(11, 1) = _covariance(11, 1) + _tmp34 + _tmp50;
+
+	_covariance(2, 2) = _bias_var(0, 0) + _covariance(11, 2) * _tmp2 + _covariance(2, 2) + _covariance(5,
+			    2) * dt + _input_var(2, 2) * _tmp0 + _tmp2 * _tmp59 + _tmp60 * dt;
+	_covariance(3, 2) = _covariance(3, 2) + _covariance(9, 2) * dt + _tmp2 * _tmp63 + _tmp61 + _tmp65 * dt;
+	_covariance(4, 2) = _covariance(10, 2) * dt + _covariance(4, 2) + _tmp2 * _tmp68 + _tmp66 + _tmp70 * dt;
+	_covariance(5, 2) = _covariance(11, 2) * dt + _covariance(5, 2) + _tmp2 * _tmp73 + _tmp71 + _tmp75 * dt;
+	_covariance(6, 2) = _covariance(6, 11) * _tmp2 + _covariance(6, 2) + _covariance(6, 5) * dt;
+	_covariance(7, 2) = _covariance(7, 11) * _tmp2 + _covariance(7, 2) + _covariance(7, 5) * dt;
+	_covariance(8, 2) = _covariance(8, 11) * _tmp2 + _covariance(8, 2) + _covariance(8, 5) * dt;
+	_covariance(9, 2) = _covariance(9, 2) + _tmp53 + _tmp64;
+	_covariance(10, 2) = _covariance(10, 2) + _tmp55 + _tmp69;
+	_covariance(11, 2) = _covariance(11, 2) + _tmp58 + _tmp74;
+
+	_covariance(3, 3) = _acc_var(0, 0) + _input_var(0, 0) * _tmp1 + _tmp15 + _tmp17 * dt;
+	_covariance(4, 3) = _input_var(1, 0) * _tmp1 + _tmp20 + _tmp22 * dt;
+	_covariance(5, 3) = _input_var(2, 0) * _tmp1 + _tmp25 + _tmp27 * dt;
+	_covariance(6, 3) = _covariance(6, 3) + _covariance(6, 9) * dt;
+	_covariance(7, 3) = _covariance(7, 3) + _covariance(7, 9) * dt;
+	_covariance(8, 3) = _covariance(8, 3) + _covariance(8, 9) * dt;
+	_covariance(9, 3) = _covariance(9, 3) + _tmp16;
+	_covariance(10, 3) = _covariance(10, 3) + _tmp21;
+	_covariance(11, 3) = _covariance(11, 3) + _tmp26;
+
+	_covariance(4, 4) = _input_var(1, 1) * _tmp1 + _tmp44 * dt + _tmp46;
+	_covariance(5, 4) = _input_var(2, 1) * _tmp1 + _tmp49 * dt + _tmp51;
+	_covariance(6, 4) = _covariance(6, 10) * dt + _covariance(6, 4);
+	_covariance(7, 4) = _covariance(7, 10) * dt + _covariance(7, 4);
+	_covariance(8, 4) = _covariance(8, 10) * dt + _covariance(8, 4);
+	_covariance(9, 4) = _covariance(9, 4) + _tmp38;
+	_covariance(10, 4) = _covariance(10, 4) + _tmp43;
+	_covariance(11, 4) = _covariance(11, 4) + _tmp48;
+
+	_covariance(5, 5) = _input_var(2, 2) * _tmp1 + _tmp73 * dt + _tmp75;
+	_covariance(6, 5) = _covariance(6, 11) * dt + _covariance(6, 5);
+	_covariance(7, 5) = _covariance(7, 11) * dt + _covariance(7, 5);
+	_covariance(8, 5) = _covariance(8, 11) * dt + _covariance(8, 5);
+	_covariance(9, 5) = _covariance(9, 5) + _tmp62;
+	_covariance(10, 5) = _covariance(10, 5) + _tmp67;
+	_covariance(11, 5) = _covariance(11, 5) + _tmp72;
+
+	_covariance(6, 6) = _bias_var(1, 1) + _covariance(6, 6);
+	// _covariance(7, 6) = _covariance(7, 6);
+	// _covariance(8, 6) = _covariance(8, 6);
+	// _covariance(9, 6) = _covariance(9, 6);
+	// _covariance(10, 6) = _covariance(10, 6);
+	// _covariance(11, 6) = _covariance(11, 6);
+
+	_covariance(7, 7) = _acc_var(1, 1) + _covariance(7, 7);
+	// _covariance(8, 7) = _covariance(8, 7);
```

Comment:
you can fill a diagonal with zeros in the derivation prevent generating this and then copy make the covariance matrix symmetric using a loop in the code

Reply:
It is now done in the auto-generated code (https://github.com/PX4/PX4-Autopilot/pull/20653/commits/dbe925698099776817ebab00d4d237da2ff21384)